app/services/place_search_service.rb

```diff
@@ -0,0 +1,14 @@
+class PlaceSearchService 
+  class << self
+    
+    def get_location_details(place_details)
```

Comment:
one thing to note is that the other endpoint on our maps api is `place_details` so we may end up changing some of this to reduce confusion. Definitely not worth holding up the merge over, though.